package at.kotlin.usermanager.exceptions

class UsernameAlreadyExistsException
    : Exception("Username already exists")